packages/components/loading/src/loading.ts

```diff
@@ -43,15 +43,18 @@ export function createLoadingComponent(options: LoadingOptionsResolved) {
       }
       removeClass(target, 'el-loading-parent--hidden')
     }
+    removeElParent()
+  }
+  function removeElParent(): void {
     vm.$el?.parentNode?.removeChild(vm.$el)
   }
-
   function close() {
     if (options.beforeClose && !options.beforeClose()) return
 
     const target = data.parent
     target.vLoadingAddClassList = undefined
     afterLeaveFlag.value = true
+    removeElParent()
```

Comment:
I don't think we should remove the child here, we should try remove in service.ts

```ts
if (resolved.fullscreen && fullscreenInstance) {
  fullscreenInstance.remvoeChild()
  fullscreenInstance.close()
}
```

Reply:
> 
OK, I accept your proposal. I will modify it immediately. Please accept my changes